airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/RouteToTaskQueueActivityImpl.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.workers.temporal.sync;
+
+import io.airbyte.config.Configs;
+import io.airbyte.workers.temporal.TemporalJobType;
+import java.util.UUID;
+
+public class RouteToTaskQueueActivityImpl implements RouteToTaskQueueActivity {
+
+  // To be replaced by proper Routing Service. For now, our MVP supports a single external Data Plane
+  private static final String MVP_DATA_PLANE_TASK_QUEUE = "MVP_DATA_PLANE";
+
+  private final Configs configs;
+
+  public RouteToTaskQueueActivityImpl(final Configs configs) {
+    this.configs = configs;
+  }
+
+  @Override
+  public String routeToTaskQueue(final UUID connectionId) {
+    if (configs.connectionIdsForDataPlane().contains(connectionId.toString())) {
```

Comment:
I know the RoutingService is still to be defined, however, I think having some of the abstraction could already help readability until we get a clearer plan.

What do you think of already defining a simple class?
```
class RoutingService {
    public RoutingService(final Configs configs) { ... }
    public String getTaskQueue(final UUID connectionId) { ... }
}
```

Moving forward, I think next steps could be transforming this into the final interface and injecting the actual implementation instead of the "naive" config based one.

Reply:
+1

Reply:
Zone of indifference for me. I think it's a good to have. We are going to start work on the Routing service almost immediately so the downside of not having is also low.

Reply:
I ended up adding this RoutingService like Jimmy suggested! See the commit here: https://github.com/airbytehq/airbyte/pull/15905/commits/706f5bb15485a5720aa4b8bc4ec3eb2feb6aeb2e